source/agg-pipeline-builder/maxtime-ms-pipeline.txt

```diff
@@ -0,0 +1,65 @@
+.. _set-maxtime-ms-agg:
+
+=======================================
+Set Max Time MS for Aggregation Queries
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the :guilabel:`Max Time MS` option on the :guilabel:`Aggregations` tab to 
+specify an upper time limit in milliseconds for aggregation pipelines
+that run in |compass|. 
+
+About This Task 
+---------------
+
+By default, :guilabel:`Max Time MS` is set to 60000 milliseconds, or 60 seconds. Consider 
+raising this value if you have a large collection or your operations frequently 
+time out.
+
+If you already specified a ``maxTimeMS`` value in the :ref:`Compass Settings 
+panel <compass-interface-settings>`, the :ref:`command line <cli-options>`, or 
+a :ref:`configuration file <config-file-options>`, the value you enter for 
+:guilabel:`Max Time MS` value must be lower than the limit you previously 
+specified.
```

Comment:
[suggestion]

As written, this is a little ambiguous as to whether we're talking about modifying Max Time MS on the agg pipeline screen, or modifying it generally. I might consider tweaking this to something like:

"... the :guilabel:`Max Time MS` you specify for your pipeline must be lower than the limit you previously specified."